File: Sakila_Net/CS_SakilaLib/Neg/NegLanguage.cs
using System.Data;
using Cl.Harrys.Sakila.DAO;
using Cl.Harrys.Sakila.Model;
namespace Cl.Harrys.Sakila.Neg
{
    public class ClNegLanguage
    {
        public static DataTable listar(int nPagina ,int cantidad)
        {
            return DAO.DAOLanguage.sqlSelectAll(nPagina, cantidad);
        }
        public  static bool consultar( ClLanguage language) 
        {
            return DAO.DAOLanguage.sqlSelect(language);
        }
        public  static int crear(ClLanguage language)
        {
            return DAO.DAOLanguage.sqlInsert(language);
        }
        public  static int eliminar(ClLanguage language)
        {
            return DAO.DAOLanguage.sqlDelete(language);
        }
        public  static int actualizar(ClLanguage language)
        {
            return DAO.DAOLanguage.sqlUpdate(language);
        }
    }
}

File: Sakila_Net/Sakila_Net/VB_SakilaLib/Neg/NegLanguage.vb
Imports  SakilaLib.Cl.Harrys.Sakila.DAO
Imports  SakilaLib.Cl.Harrys.Sakila.Model
Namespace Cl
    Namespace Harrys
        Namespace Sakila
            Namespace Neg

                Public Class ClNegLanguage
                    Public Shared Function listar(nPagina as Integer ,cantidad as Integer) as DataTable
                        Return DAO.DAOLanguage.SelectAll(nPagina, cantidad)	   
                    End Function
                    Public  Shared Function consultar( language as ClLanguage) as Boolean
                        Return DAO.DAOLanguage.SelectOne(language)     
                    End Function
                    Public  Shared Function crear( language as ClLanguage) as Integer
                        Return DAO.DAOLanguage.SqlInsert(language)     
                    End Function
                    Public  Shared Function eliminar( language as ClLanguage) as Integer
                        Return DAO.DAOLanguage.SqlDelete(language)     
                    End Function
                    Public  Shared Function actualizar( language as ClLanguage) as Integer
                        Return DAO.DAOLanguage.SqlUpdate(language)     
                    End Function
                End Class

            End Namespace  
        End Namespace  
    End Namespace  
End Namespace  
